longevity_test.py

```diff
@@ -105,6 +108,25 @@ def test_custom_time(self):
 
         self.run_prepare_write_cmd()
 
+        # Grow cluster to target size if requested
```

Comment:
At the moment this part is taking place after the prepare (so we have some data in the cluster while growing) but without any stress.
On one hand, it's better to test the grow under stress, but OTOH it may take too long so the stress will complete before the cluster grew to the target size and before any nemesis has run or not enough nemesis.

Hence, I prefer to leave it like this for now and see how it goes.